aws/resource_aws_ssm_maintenance_window_task.go

```diff
@@ -190,23 +202,31 @@ func resourceAwsSsmMaintenanceWindowTask() *schema.Resource {
 										Elem: &schema.Resource{
 											Schema: map[string]*schema.Schema{
 												"notification_arn": {
-													Type:     schema.TypeString,
-													Optional: true,
+													Type:         schema.TypeString,
+													Optional:     true,
+													ValidateFunc: validateArn,
 												},
 
 												"notification_events": {
 													Type:     schema.TypeList,
 													Optional: true,
-													Elem:     &schema.Schema{Type: schema.TypeString},
+													Elem: &schema.Schema{
+														Type: schema.TypeString,
+														ValidateFunc: validation.StringInSlice([]string{
+															"All",
+															"InProgress",
+															"Success",
+															"TimedOut",
+															"Cancelled",
+															"Failed",
+														}, false),
```

Comment:
Looks like the enumeration is now available:

```suggestion
														Type:         schema.TypeString,
														ValidateFunc: validation.StringInSlice(ssm.NotificationEvent_Values(), false),
```